src/client/components/household-information/ChildIncome.jsx

```diff
@@ -12,26 +12,39 @@ import { isBlank, isValidMonetaryValue, validateIfDirty } from '../../utils/vali
  * `onValueChange` - a function with this prototype: (newValue)
  */
 class ChildIncome extends React.Component {
+
+  getRequiredErrorMessage(field) {
+    let result;
+    if (isValidField(isValidMonetaryValue, field)) {
+      if (field.dirty && field.value === '') {
```

Comment:
Done--  This was a little awkward because we don't really have a way to chain multiple validations to different error messages.  It does seem more readable and DRY now though.